fonts/mscore/genttf

```diff
@@ -1,6 +1,13 @@
-#!/usr/bin/fontforge -lang=ff
-
-Open($1)
-Generate($1:r + ".ttf")
+#!/bin/sh
 
+# ensure that user preferences don't leak into the fonts
+FONTFORGE_LOADPREFS=Never
+FONTFORGE_VERBOSE=1
+LC_ALL=C
+export FONTFORGE_LOADPREFS FONTFORGE_VERBOSE LC_ALL
 
+# batch convert, with options suitably set
+fontforge -lang=ff -script - "$1" <<'EOF'
+Open($1)
+Generate($1:r + ".ttf", "", 0x800)
```

Comment:
It would be possibly useful to document what the `0x800` parameter means, without forcing the reader to peruse the FF script documentation which is not easy to discover.

Reply:
@mirabilos isn't it a good time to add a comment? :)